### ISSUE TITLE
<bug title: summarize bug in one line> a11y: CodeMirror colour palette

### ISSUE THREAD

**Issue by edenhernandez-suse** [rancher/dashboard#14668]
**Context**
Our code blocks use `CodeMirror` with a set of colours that should be improved.

**Problem**
We identified several accessibility/usability issues with the colour palette:
- It's the same for both Light and Dark modes. This means that clear colours don't work on light backgrounds, and darker colours fail on dark backgrounds.
  - Boolean values and comments fail on dark backgrounds
  - String values fail on light backgrounds
- Number values take the same colour as keys/props and are not properly differentiated

![Image](https://github.com/user-attachments/assets/8730fe15-338b-4beb-a0cd-e853bd7415a5)

![Image](https://github.com/user-attachments/assets/212cc730-e3c2-4ff4-9027-d2e77b9b1b15)

![Image](https://github.com/user-attachments/assets/4ade8324-c90c-437b-8a4e-93e77269e2e3)

![Image](https://github.com/user-attachments/assets/cd14d4fb-1b1c-4ed0-97d3-0a91af4a2e32)

**Solution**
***Light mode colour palette:***

![Image](https://github.com/user-attachments/assets/8f2392b9-65d4-4aa1-882f-7f89cb4d2c8f)

`plain-text, .cm-meta: var(--body-text)`
`.cm-string:  #00529F`
`.cm-number:  #005E3B`
`.cm-keyword: #B94545`
`.cm-atom:    #921E80`
`.cm-comment: #8F5536`

![Image](https://github.com/user-attachments/assets/2a7449be-3aa0-4975-babf-46035c06edd0)

***Dark mode colour palette:***
TBD

**Comment by nwmac** [comment]
@edenhernandez-suse Are you really planning this for 2.12.0?

**Comment by edenhernandez-suse** [comment]
> [@edenhernandez-suse](https://github.com/edenhernandez-suse) Are you really planning this for 2.12.0?

No, sorry, it was a mistake.

**Comment by codyrancher** [comment]
You may want to consider also resolving this https://github.com/rancher/dashboard/issues/5485 at the same time.
